pages/error.js

```diff
@@ -10,7 +10,7 @@ export default function ErrorPage(props) {
   return (
     <div className="min-h-screen relative">
       <Head>
-        <title>Error - Erreur</title>
+        <title data-gc-analytics-error={props.statusCode}>Error - Erreur</title>
```

Comment:
The status code isn't necessarily always going to be there, but I'm also not exactly sure what you could put here.